packages/dashboard/src/components/global-shortcuts/global-shortcuts.tsx

```diff
@@ -0,0 +1,295 @@
+    return params.orgId;
+  };
+
+  // Detect current context for Create shortcut
+  const getCurrentContext = () => {
+    const path = location.pathname;
+    if (path.includes('/content/playlists')) return 'playlists';
+    if (path.includes('/content/medias')) return 'medias';
+    if (path.includes('/channels')) return 'channels';
+    if (path.includes('/devices')) return 'devices';
+    if (path.includes('/teams')) return 'teams';
+    return null;
+  };
+
+  const handleCreate = () => {
+    const context = getCurrentContext();
+    if (!context) return;
+
+    // Trigger click on the create/add/upload button in the current page
+    // Strategy: Find buttons by common patterns in text content (works across languages)
+    const buttons = Array.from(document.querySelectorAll('button'));
+
+    const actionButton = buttons.find((btn) => {
+      const text = btn.textContent?.toLowerCase() || '';
+      const ariaLabel = btn.getAttribute('aria-label')?.toLowerCase() || '';
+
+      // Check for common action words from actual translations
+      const actionWords = [
+        // English
+        'add',
+        'create',
+        'new',
+        'upload',
+        'register',
+        // Spanish
+        'añadir',
+        'agregar',
+        'crear',
+        'nuevo',
+        'subir',
+        'registrar',
+        // Swedish
+        'lägg till',
+        'skapa',
+        'ny',
+        'ladda upp',
+        'registrera',
+        // German
+        'hinzufügen',
+        'erstellen',
+        'neu',
+        'hochladen',
+        'registrieren',
+        // French
+        'ajouter',
+        'créer',
+        'nouveau',
+        'télécharger',
+        'enregistrer',
+        // Chinese
+        '添加',
+        '创建',
+        '新',
+        '上传',
+        '注册',
+        // Arabic
+        'إضافة',
+        'إنشاء',
+        'جديد',
+        'رفع',
+        'تسجيل',
+        // Korean
+        '추가',
+        '만들기',
+        '새',
+        '업로드',
+        '등록',
+        // Japanese
+        '追加',
+        '作成',
+        '新しい',
+        'アップロード',
+        '登録',
+      ];
+
+      return actionWords.some(
+        (word) => text.includes(word) || ariaLabel.includes(word)
+      );
+    });
+
+    if (actionButton) {
+      (actionButton as HTMLButtonElement).click();
+    } else if (props.onCreateResource) {
+      props.onCreateResource();
+    }
+  };
+
+  onMount(() => {
+    // Show shortcuts legend - using ? which is shift+/ on most keyboards
+    registerShortcut('show-shortcuts', {
+      key: '?',
+      shift: true,
+      description: () => t('shortcuts.showShortcutsLegend'),
+      category: 'global',
+      action: () => props.onShowShortcuts(),
+    });
+
+    // Global Search - moved to global category
+    registerShortcut('global-search', {
+      key: 'F',
+      ctrl: true,
+      description: () => t('shortcuts.search'),
+      category: 'global',
+      action: () => {
+        const searchInput = document.querySelector(
```

Comment:
This is a hack, we need a generic way to activate the global search that do not depend on jQuery style code.